modules/ROOT/pages/slsa.adoc

```diff
@@ -0,0 +1,130 @@
+a| This rule verifies that the build task was called with a particular parameter specifying the build should be done hermeticly. This rule is specific
+to RHTAP's task definitions, since EC isn't able to explicitly confirm that the build was indeed hermetic. But, when combined with the strictest
+"acceptable task bundles" rule, and a trustable source for the task definition, we can use the rule to ensure that only builds performed hermeticly can
+be released.
+
+a| https://slsa.dev/spec/v1.0/verifying-artifacts#step-1-check-slsa-build-level[Verifying Artifacts (v1.0)]
+a| xref:ec-policies:ROOT:release_policy.adoc#slsa_build_build_service__slsa_builder_id_accepted[SLSA Builder ID is known and accepted]
+a| The verification process described in the SLSA v1.0 spec includes a check to confirm the builder id matches what is
+expected. This rule will fail if an unexpected builder id is found in the SLSA attestation. (For RHTAP currently the builder id
+is the default used by Tekton Chains, though this may change in the future.)
+
+a| https://slsa.dev/spec/v1.0/threats#f-upload-modified-package[Threat (G) - Upload modified package],
+https://slsa.dev/spec/v1.0/threats#h-use-compromised-package[Threat (H) - Use compromised package] (v1.0)
+a| Image signature check passed
+a| Enterprise Contract uses a public key to validate the signature of the image using https://docs.sigstore.dev/cosign/overview/[Cosign]. If the image
+signature can't be verified for any reason, a failure will be reported. There are other ways to verify image signatures, e.g. by using `cosign` directly,
+but EC will happily do it for you.
+
+There is also experimental support in EC for validating signatures for images signed keylessly
+using https://docs.sigstore.dev/fulcio/overview/[Sigstore Fulcio], though RHTAP currently uses long-lived
+signing keys to sign images and attestations.
+
+a| https://slsa.dev/spec/v1.0/verifying-artifacts[Verifying Artifacts (v1.0)]
+a| Attestation signature check passed
+a| As well as verifying the image signature, EC also verifies the signature of the attestation, which is an important part of the artifact
+verification process. For RHTAP the same key is used to sign the attestation and the image itself. As mentioned above, there is experimental
+support for verifying keylessly signed attestations.
+
+a| https://slsa.dev/spec/v1.0/verifying-artifacts#step-1-check-slsa-build-level[Verifying Artifacts (v1.0)]
+a| xref:ec-policies:ROOT:release_policy.adoc#slsa_build_scripted_build__subject_build_task_matches[Provenance subject matches build task image result]
+a| SLSA recommends an artifact verifier checks that the that provenance statement’s subject matches the digest of the artifact in question.
+This is a good sanity check to confirm that the provenance being considered is for expected image. Because EC uses Cosign under the hood, and
```

Comment:
```suggestion
is for the expected image.
```

Reply:
It may be worth considering if there's a different way to phrase "sanity check" here. IBM's style guide forbids using that phrase in any Red Hat documentation. Not sure if that applies to upstream documentation too. See https://issues.redhat.com/browse/STONEINTG-233 for when the Integration team renamed their tasks.

Reply:
Ack, thanks. Changed it to "performs and additional confirmation".